public/sass/components/_empty_list_cta.scss

```diff
@@ -1,24 +0,0 @@
-.empty-list-cta {
```

Comment:
this class is being used 4-5 other locations as well (not only from this react component), templating list, annotation links, dashboard links, manage dashboard, notifications list, playlists. 

Not sure why the react componet (and angular wrapper) is not being used, think for the annoations & templating use cases it did not work as we need custom body with help box but for the others not sure.

Reply:
![Screenshot 2019-03-27 at 08 34 48](https://user-images.githubusercontent.com/23470681/55058053-35739200-506b-11e9-96bf-0c63e4d6ffd4.png)

Reply:
> Not sure why the react componet (and angular wrapper) is not being used, think for the annoations & templating use cases it did not work as we need custom body with help box but for the others not sure.

@torkelo the new component accepts `message`/`footer` props that can be a React elements so we can achieve than

Reply:
@torkelo created followup issue as this is not in the scope of this PR #16248. Also, brought the sass file back untill #16248 is done.

Reply:
@Ijin08 could you please re-check this?